packages/toolpad-core/src/types/Navigation.types.ts

```diff
@@ -0,0 +1,13 @@
+export interface NavigationRoute {
```

Comment:
I think I prefer to keep the word "route" out of the nomenclature, this is not a router